inf-clojure.el

```diff
@@ -238,11 +238,23 @@ This should usually be a combination of `inf-clojure-prompt' and
 `inf-clojure-subprompt'."
   :type 'regexp)
 
-(defcustom inf-clojure-prompt-on-set-ns t
+(defcustom inf-clojure-prompt-on-commands t
   "Controls whether to prompt when switching namespace."
   :type '(choice (const :tag "always" t)
                  (const :tag "never" nil))
-  :package-version '(inf-clojure . "2.0.0"))
+  :package-version '(inf-clojure . "2.0.0")
+  :safe 'booleanp)
+
+(defun inf-clojure--prompt-on-commands-p (&optional arg)
```

Comment:
Isn't copying this an overkill? It was needed in CIDER, because some commands had prefix arguments of their own, but here it seems pointless to make it this complex.

Frankly I wonder if we even need this `defcustom` we can make all the commands not prompt by default and to prompt with a prefix argument (or when there's nothing at point). It would be simple and would get the job done.